### DISCUSSION TITLE
Problem running Firejail as another user

### DISCUSSION THREAD

**Discussion by Newrad-G** [netblue30/firejail#5135] [Q&A]
Hello,

I am attempting to successfully run firejail as another user (who also has admin priveleges) with:
_sudo -u otheruser firejail chromium %U_

For simplicty I have kept the profile the same (as main user) and only allow access to the ${DOWNLOADS} folder.

With firejail as loggedin/main user for Chromium everything works fine.

With firejail as otheruser everything works fine - except that when I download to a destination not allowed, say to "Documents", _**the download is allowed into "Documents".**_

So I have looked at the difference in messages for otheruser in the terminal. These are:

**At Launch of Chromium 2 extra messages of the type:**
_[4:41:0508/144902.437597:ERROR:bus.cc(398)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")_

_libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)_

**When starting Download of a document extra messages almost same as below appear:**
_• (chromium:4): dconf-WARNING **: 12:44:38.723: failed to commit changes to dconf: Unknown or unsupported transport “disabled” for address “disabled:”_

**When clicking Chromium for "Show in folder", extra messages are similar to:**
_[4:4:0508/150147.221952:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.NameHasOwner: object_path= /org/freedesktop/DBus: unknown error type: 
[4:4:0508/150147.221983:ERROR:platform_util_linux.cc(138)] Failed to call NameHasOwner_

_[4:4:0508/150147.224501:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.ListActivatableNames: object_path= /org/freedesktop/DBus: unknown error type: 
[4:4:0508/150147.224533:ERROR:platform_util_linux.cc(179)] Failed to call ListActivatableNames_

_** (org.gnome.Nautilus:343): WARNING **: 15:01:47.333: Error on getting connection: Failed to load SPARQL backend: Unable to find default domain ontology rule /usr/share/tracker/domain-ontologies/default.rule
(org.gnome.Nautilus:343): dconf-WARNING **: 15:01:47.728: failed to commit changes to dconf: Unknown or unsupported transport “disabled” for address “disabled:”
** (org.gnome.Nautilus:343): CRITICAL **: 15:01:47.829: update_dbus_opened_locations: assertion 'dbus_object_path' failed
(org.gnome.Nautilus:343): GLib-GIO-CRITICAL **: 15:01:53.039: g_dbus_proxy_new_sync: assertion 'G_IS_DBUS_CONNECTION (connection)' failed_

Is the information I have given enough to diagnose the problem?

I am just marginally techie, and relatively new to firejail. I have explored above messages on the web but with little successful insight.

Thank you very much for any help on this.

[System is Q4OS (Debian11-based); Desktop is Gnome 3.38]

**Comment by rusty-snake** [comment]
> sudo -u otheruser firejail chromium %U

1. Why the `%U`? Neither `sudo` nor `firejail` nor `chromium` can do something useful with it.
2. Why do you want to run chromium as an other user? What are you trying to solve? Is there a better way?

> Failed to connect to the bus
> failed to commit changes to dconf:
> Failed to call method: 

So otheruser has no running session, right? Running a program like chromium is always better in a session. Here we are at 2. from above.
